### ISSUE TITLE
Quest Handheld Rock Smasher not finishing, not detecting Biodiesel bucket

### ISSUE THREAD

**Issue by RobbeSchepens** [superfluke/undiscovered#73]
Updated to version 13 today. I have gotten a bucket of diesel from the Refinery yet this quest does not want to finish and detect the bucket. The drill and head are marked as COMPLETE but not the bucket. I tried spawning in a bucket or putting it in my hand to see if it works, but no progress. 

[Screenshot of biodiesel bucket in my inventory](https://i.imgur.com/NVZF26S.png)

Edit: I relogged and spawned in a bucket, that completed the quest.

**Comment by superfluke** [comment]
Yeah I'm getting the same thing now. Seems to randomly affect some quests that require buckets. Logging out then logging back and hitting the 'Detect/Submit' quest button seems to solve it. I'll leave this open for now and see if I can find out why it has started acting up.
